cmake/SSGCommon.cmake

```diff
@@ -744,7 +753,7 @@ macro(ssg_build_product PRODUCT)
         DESTINATION "${SSG_CONTENT_INSTALL_DIR}")
     install(FILES "${CMAKE_BINARY_DIR}/ssg-${PRODUCT}-ds.xml"
         DESTINATION "${SSG_CONTENT_INSTALL_DIR}")
-    install(FILES "${CMAKE_BINARY_DIR}/ssg-${PRODUCT}-cpe-dictionary.xml"
+    install(FILES "${CMAKE_BINARY_DIR}/ssg-cpe-dictionary.xml"
```

Comment:
Shouldn't `ssg_build_cpe_dictionary` install the dictionary instead of doing it every time a product is built?